create_scavenger_hunt.sh

```diff
@@ -41,7 +38,18 @@ echo "less, head, tail" >> ${PATH_LEVEL_3}/README.txt
 
 # Level 5
 PASSWORD5="needle"
-curl -s -X POST https://lipsum.com/feed/json -d "amount=3000" | sed -e "s/.\{100\}/&\n/g" > ${PATH_LEVEL_4}/password_raw.txt
+retry_command ()
+{
+    for i in 1 2 3 4 5; do
+        if [[ $i -eq 5 ]]; then
+            "$@"
+        else
+            "$@" && break || sleep $((i * 5))
+        fi
+    done
+}
+# The jq will reformat the json a bit, but its main purpose is to check that curl returned valid json (no server error).
+retry_command curl -s -X POST https://lipsum.com/feed/json -d "amount=3000" | jq . | sed -e "s/.\{100\}/&\n/g" > ${PATH_LEVEL_4}/password_raw.txt
```

Comment:
On the CI, some tests (solving) spuriously failed:
* level5: `password: ""`
* level15: `password: "ac"`

I guess sometimes some error-html page is returned instead?